ui/app/components/clients/activity.ts

```diff
@@ -65,18 +65,18 @@ export default class ClientsActivityComponent extends Component<Args> {
       return activity.byMonth;
     }
     const namespaceData = activity.byMonth
-      .map((m) => m.namespaces_by_key[namespace as keyof typeof m.namespaces_by_key])
+      .map((m) => m.namespaces_by_key[namespace])
       .filter((d) => d !== undefined);
 
     if (!mountPath) {
-      return namespaceData.length === 0 ? undefined : namespaceData;
+      return !namespaceData ? [] : namespaceData;
     }
 
-    const mountData = mountPath
```

Comment:
if there isn't a mountPath we hit the block above, so no need for the ternary here